deploy/role_binding.yaml

```diff
@@ -9,3 +9,17 @@ roleRef:
   kind: Role
   name: apicurio-operator
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
It needs this role due to ImageStream object creation from the template.

Reply:
is there no specific role for image streams? cluster-admin seems a bit strong